.github/workflows/benchmark-command.yml

```diff
@@ -41,6 +41,12 @@ jobs:
       - name: Run Benchmarks
         run: earthly --verbose -P +benchmark
 
+      - name: Fetch previous results
+        run: bash ./scripts/clone-gh-pages.bash
+
+      - name: Refresh Flink Benchmark
```

Comment:
Github actions can be run weekly/monthly using some cronjob like syntax, so instead of doing this every time, I think it's better to make a separate benchmark-flink.yml file that where you say "run this every month"